cmake/LYTestWrappers.cmake

```diff
@@ -415,6 +418,7 @@ function(ly_add_googletest)
         NON_IDE_PARAMS ${non_ide_params}
         RUNTIME_DEPENDENCIES AZ::AzTestRunner
         COMPONENT ${ly_add_googletest_COMPONENT}
+        TEST_SERIAL
```

Comment:
this will make every test serial, here you should pass `${ly_add_googletest_TEST_SERIAL}` to consume the parameter